src/styles/index.scss

```diff
@@ -1,3 +1,258 @@
-body {
+@import 'variables';
+
+body,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: $fw-regular;
```

Comment:
Ensure that the variable `$fw-regular` is defined in the 'variables' file. If not, it may cause a compilation error.